src/modules/sub-items/sub.items.module.js

```diff
@@ -568,6 +585,34 @@ export default class SubItemsModule extends Component {
         this.bulkDelete();
     }
 
+    handleBulkOperationFailedNotification(selectedItems, failedLocations, modalTitle, notificationMessage) {
+        const { contentTypesMap } = this.state;
+        const failedItems = failedLocations.map(({ id: locationId }) => selectedItems.get(locationId));
+        const failedItemsData = failedItems.map((item) => {
+            const contentType = contentTypesMap[item.content.ContentType._href];
+            const contentTypeIdentifier = contentType.identifier;
+            const contentTypeName = window.eZ.adminUiConfig.contentTypeNames[contentTypeIdentifier];
+
+            return {
+                contentTypeName,
+                contentName: item.content.Name,
+            };
+        });
+
+        window.eZ.helpers.notification.showWarningNotification(notificationMessage, (notificationNode) => {
+            const showModalBtn = notificationNode.querySelector('.ez-notification-btn--show-modal');
+
+            if (!showModalBtn) {
+                return;
+            }
+
+            showModalBtn.dataset.modalTitle = modalTitle;
+            showModalBtn.dataset.failedItemsData = JSON.stringify(failedItemsData);
```

Comment:
I think there is no point to adding this to a DOM element and getting this after clicking on the button.

You can pass it directly to the prop `onShowBulkOperationFailedModal`

Reply:
Hmm, you're right, we can bind these data to `this.props.onShowBulkOperationFailedModal` function. 👍